doc-content/enterprise-only/installation/patches-applying-proc.adoc

```diff
@@ -109,15 +109,6 @@ The update script creates a `backup` folder in the extracted `{PRODUCT_INIT}-$VE
 . After the update tool completes, return to the *Software Downloads* page of the Red Hat Customer Portal where you downloaded the update tool and install any other updates or new release artifacts that are relevant to your {PRODUCT} distribution.
 +
 For files that already exist in your {PRODUCT} distribution, such as `.jar` files for the {DECISION_ENGINE} or other add-ons, replace the existing version of the file with the new version from the Red Hat Customer Portal.
-. If you use the standalone *{PRODUCT} {PRODUCT_VERSION_LONG} Maven Repository* artifact (`{PRODUCT_FILE}-maven-repository.zip`), such as in air-gap environments, download *{PRODUCT} [VERSION] Incremental Maven Repository* and extract the downloaded `{PRODUCT_INIT}-$VERSION-incremental-maven-repository.zip` file to your existing `~/maven-repository` directory to update the relevant contents.
```

Comment:
In fact, this step should be kept in the document, the user only has to use the full Maven repository instead of the incremental Maven repository. 

The need to add Maven artifacts to the offline Maven repository is still valid, we just dropped the incremental repository zip, so users have to use the full repository zip as the source of new artifacts. When they simply extract it into their existing Maven repository directory, they may be prompted to confirm overwriting of some files, since some of the files might already be there.